### DISCUSSION TITLE
read_csv ignore quotes option?

### DISCUSSION THREAD

**Discussion by eli-daniels** [duckdb/duckdb#13259] [Q&A]
Hello,
Thanks for this amazing tool! I've looked through the documentation and issues but cannot find an answer to my issue.

I've run into an issue reading csv files that are not very clean. They contain lone quotes between delimiters. For example : he'llo, h"ello, hello, etc...
Comma being the delimiter in this case.

This issue could be resolved if the quote argument for read_csv could be ignored entirely and have all quotes treaded as simple characters. Is this possible ?

Cheers,
Eli

**Comment by pdet** [comment]
Maybe `quote= ''`?

**Reply by eli-daniels** [reply]
The value of quote must have bit length 1. At least that is the error I get when trying `quote= ''`

**Reply by pdet** [reply]
Can you share the full query?
Afaik, this should work:
```sql
FROM read_csv('path/to/file.csv',  quote='');

```
